algorithms.tex

```diff
@@ -1,2 +1,17 @@
+% A description of some of the algorithms in SymPy. The list is not
+% exhaustive.
 
+% The sections here are preliminary. We may end up needing to cut some of
+% this.
 
+\subsection{Numerics}
+
+\subsection{Polynomials}
+
+\subsection{The Risch Algorithm}
+
+\subsection{The Gruntz Algorithm}
+
+\subsection{Logic}
+
+\subsection{Other}
```

Comment:
That's a good point. I'm not worried about organizational issues right now. My main concern is getting the stuff written. Then we can arrange it. But you're probably right that it would be better to start with the generic list and then have specific sections on things we want to expand on.